poscreators/integration-checklist.md

```diff
@@ -0,0 +1,22 @@
+| ![All markets](images/flag-eu.png) | **Closing receipts are implemented** | Daily, monthly and yearly closing receipts are implemented via the respective ´ftRecreiptCases`. | [Receipt cases](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/general/reference-tables) & relevant country-specific appendices |
+| ![All markets](images/flag-eu.png) | **Error handling is implemented** | The different types of errors (Middleware not reachable & signing device/service not reachable) are properly handled by the POS software | [Cash register integration](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/general/cash-register-integration) & relevant country-specific appendices |
+| ![All markets](images/flag-eu.png) | **Signatures are printed onto the physical receipts** | The `ftSignatureItems` returned by the Middleware are properly printed onto the physical receipts. Please note that generally all items need to printed, with some exceptions in Germany (more details available in the linked docs). | [Reference tables](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/general/reference-tables#type-of-signature-ftsignaturetype) & relevant country-specific appendices |
+| ![All markets](images/flag-eu.png) | **Production Portal account created** | An account in the production Portal was created to manage POS Systems. | [Getting Started Guide](README.md) |
+| ![All markets](images/flag-eu.png) | **POS Systems are created in the production Portal** | One POS System per model (not per actual device) is created in the Portal, and its ID is used as `cbPosSystemID` in sign requests. | [Getting Started Guide](README.md) |
+| ![All markets](images/flag-eu.png) | **POS Dealers are invited and connected to POS Systems** | The POS Dealers distributing the device are either invited by their POS Creator or created their accounts themselves, and are connected to the previously created POS System. | [Getting Started Guide](README.md) |
+| ![All markets](images/flag-eu.png) | **_(Optional):_ Correct usage of business cases is approved by tax consultant** | A tax consultant who is familiar with the business sequences that are implemented via the Middleware approves the correct usage of _receipt-_, _charge item-_ and _pay item cases_. | [Getting Started Guide](README.md) |
+| ![Germany](images/flag-de.png) | **_(Optional):_ DSFinV-K export is approved by tax consultant** | A tax consultant who is familiar with the business sequences that are implemented via the Middleware approves the content of a sample or live-data DSFinV-K export created via the Middleware or the Portal (Please note that structural integrity is [already certified](https://fiskaltrust.de/fiskaltrust-middleware-dsfinv-k-schnittstelle-jetzt-gobd-zertifiziert/)). | [Getting Started Guide](README.md) |
```

Comment:
France:
Certification or Attestation has to be done.
Certification needs a contract and more following steps.
Attestation needs the implementation and the signed letter from PosCreator and fiskaltrust.

Reply:
Valid point, we also thought about adding it. Which address would a PosCreator need to contact regarding the certification? Via the [contact formular](https://fiskaltrust.fr/en/contact-us/)?

Reply:
I think email at contact@fiskaltrust.fr should do the thing.

Reply:
We've added one more point regarding this, thanks again 🙂